.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,23 +1,28 @@
 <!--
 Thanks for sending a pull request!
 
-Please make sure you've read our contributing guidelines and how to submit a pull request (https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests).
-
-Finally, be sure to sign commits with your real name. Since by opening
-a PR you already have commits, you can add signatures if needed with
-something like `git commit -s --amend`.
+For more detailed information, please review our contributing guidelines:
+https://github.com/containers/podman/blob/main/CONTRIBUTING.md#submitting-pull-requests
 -->
 
+#### Checklist
+
+Ensure you have completed the following checklist for your pull request to be reviewed:
+
+- [ ] Commits are signed with my real name (`git commit -s`). If needed, use (`git commit -s --amend`).
+- [ ] Tests have been added/updated (or no tests are needed)
+- [ ] Documentation has been updated (or no documentation changes are needed)
+- [ ] All commits pass `make validatepr` (format/lint checks)
+- [ ] Referenced issues using `Fixes: #00000` in commit message (if applicable)
```

Comment:
It might be better to move this item after the signing commits, as that will group it with other commit operations.

Reply:
done